packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -571,15 +576,15 @@ export function TransferPanelMain({
     }.`
   }, [errorMessage, isDepositMode, openWithdrawOnlyDialog])
 
-  function switchNetworks() {
+  const switchNetworks = useCallback(() => {
```

Comment:
We have both functions `switchNetwork` and `switchNetworks` doing different things in the same file. It's confusing to read. We should clean it up a little bit. Probably rename them to their exact-verbose purpose.

Reply:
renamed